### PR TITLE
generate a db name when one is not provided

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: src/services/iasql.ts [hunk 1]
```diff
@@ -24,13 +24,20 @@ export function recordCount(records: { [key: string]: any, }[]): [number, number
   return [ dbCount, cloudCount, bothCount, ];
 }
 
+// config for unique name generator
+// e.g. big_red_donkey
```
Comment (dfellis) [inline src/services/iasql.ts:28]: purple monkey dishwasher

**Review by dfellis — APPROVED** [review]
I don't see a problem with this, but couldn't the CLI do this and not require a change in both?

**Comment by depombo** [comment]
> I don't see a problem with this, but couldn't the CLI do this and not require a change in both?

I did this in case we ever need to do name reservation (right now it just needs to be unique per user account) and check other requirements Postgres/IP might have for the name. We did something similar with any cloud because azure was super picking about resource names and so on.

File: src/services/iasql.ts (hunk 1 above)
Comment (depombo) [reply src/services/iasql.ts:28]: 😂

**Pull request merged**